python/ray/tests/test_webui.py

```diff
@@ -1,50 +0,0 @@
-import re
-import sys
-import time
-
-import pytest
-import requests
-
-import ray
```

Comment:
We probably need to delete this from BUILD.bazel. Same for test_metrics. Also, this test is run as a sanity check for the wheel builds. I think we can do the same thing for the new dashboard.

Reply:
I removed both of them from BUILD.bazel
As far as running it as a sanity check for the wheels, that makes sense. I was having trouble running a test inside the dashboard directory in this case. There was an import error importing things from conftest. Given how long the cycle time is in trying to fix something like that over CI, I decided it probably wasn't worth the effort. What do you think?